src/main/java/game/baseball/player/ComputerPlayer.java

```diff
@@ -0,0 +1,25 @@
+package game.baseball.player;
+
+import game.baseball.ball.Balls;
+import game.baseball.util.number.NumberGenerator;
+
+import static game.baseball.util.setting.GameSetting.GAME_SETTING;
+
+public class ComputerPlayer implements Player {
+
+    private Balls balls;
+    private final NumberGenerator numberGenerator;
```

Comment:
```suggestion
    private final NumberGenerator numberGenerator;
    private Balls balls;
```

이건 관습적이긴한데 순서가 바뀌어야 겠군요.!